content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,17 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
+
+To make the resource quota work on ephemeral-storage, two things need to be done.
+
+1.admin set the ephemeral-storage resource quota limit
+2.User needs to mention the ephemeral-storage resource quota limit in pod spec.
+
+If the user doesn't specify the ephemeral-storage resource limit in pod spec, then the resource quota is not enforced on ephemeral-storage. For example, if the admin set the ephemeral-storage quota and if the user doesn't specify the ephemeral-storage limit in the pod creation spec then the  resource quota on ephemeral-storage will not be enforced. Thus the user can use n number of ephemeral storage.
```

Comment:
Please wrap the long line; please fix grammar of the last sentence.

Reply:
@tengqm i gave a try to wrap it up. Please take a look and let me know if this sounds ok to you.
Thanks!!